pkg/plugins/config/config.go

```diff
@@ -31,6 +31,8 @@ type Cfg struct {
 	LogDatasourceRequests bool
 
 	PluginsCDNURLTemplate string
+
+	AppURL string
```

Comment:
Nit:
```suggestion
	GrafanaAppURL string
```

Might help to be more explicit here when treating this as part of the public plugins API